content/problems/bio1/index.tsx

```diff
@@ -12,7 +12,7 @@ export const bio1Problems = {
     solutionAuthors: ["Alex Pylypenko"],
     component: <IntegerStrings />,
   } as ProblemInfo,
```

Comment:
I think if we make it an array, we can specify the type of the array like this:

`export const bio1Problems: ProblemInfo[] = [...];`

and then we won't have to write `as ProblemInfo` after each array entry. Can you try this and see if it works?